staging/src/k8s.io/code-generator/cmd/client-gen/generators/util/tags.go

```diff
@@ -146,16 +242,96 @@ func ParseClientGenTags(lines []string) (Tags, error) {
 		}
 		ret.SkipVerbs = skipVerbs
 	}
+	var err error
+	if ret.Extensions, err = parseClientExtensions(values); err != nil {
+		return ret, err
+	}
 	return ret, validateClientGenTags(values)
 }
 
+func parseClientExtensions(tags map[string][]string) ([]extension, error) {
+	var ret []extension
+	for name, values := range tags {
+		if !strings.HasPrefix(name, genClientPrefix+"method") {
+			continue
+		}
+		for _, value := range values {
+			// the value comes in this form: "Foo,verb=create"
+			ext := extension{}
+			parts := strings.Split(value, ",")
+			if len(parts) == 0 {
+				return nil, fmt.Errorf("invalid of empty extension verb name: %q", value)
+			}
+			// The first part represents the name of the extension
+			ext.VerbName = parts[0]
+			if len(ext.VerbName) == 0 {
+				return nil, fmt.Errorf("must specify a verb name (// +genclient:method=Foo,verb=create)")
+			}
+			// Other parts are additional configuration (verb=TYPE is required)
+			params := parts[1:]
+			for _, p := range params {
+				parts := strings.Split(p, "=")
```

Comment:
This looks like a helper for parsing a token.  Nothing else needed this?

Reply:
there is no other generator command that takes extra arguments (onlyVerbs, skipVerbs are all main tokens), so nothing in generator AFAIK.